node-website-feed/README.md

```diff
@@ -11,3 +13,147 @@ Bare bones off the shelf website feed for NodeJS, including Client Credentials a
+- The `propertyId` values (represented as foo, bar & baz), are the ids of the properties in your DB. You should break the query into batches of 100 propertyIds at a time to avoid timeouts and use paging to return all images changed.
+
+```ts
+'https://platform.reapit.cloud/propertyImages?propertyId=foo&propertyId=bar&propertyId=baz&pageSize=100&pageNumber=1&modifiedFrom=2022-05-26T10:38:06.581Z&modifiedTo=2022-05-26T11:38:06.581Z'
+```
+
+In both cases, when you have received your updated properties, you should update the corresponding records in your database.
+
+#### 3. **Housekeeping**
+
+Each day (or on a schedule of your choosing), we would recommend checking that your property data has not gone stale. This can be done by fetching each property by id in batches of 100 using the below query where the ids of the properties in your database are represented by foo, bar and baz.
+
+```ts
+'https://platform.reapit.cloud/properties?id=foo&id=bar&id=baz'
+```
+
+On fetching of this request, you should validate that the returned data is still valid i.e. it matches your `forSale`, `toLet` filters and so on. Records that are no longer valid should be remove from your database.
+
+### Estimating Charges
+
+You will be billed monthly for using the foundations API on a consumption model, per API call. The cost per call is on a sliding scale, getting cheaper the more calls you make. [see here](https://foundations-documentation.reapit.cloud/developer-terms-and-conditions#schedule-2-fees) for details of these changes.
+
+Consumption may vary significantly from customer to customer, depending on how they use Reapit Software however, we have attempted to give you an illustrative example, based on real world numbers from an SME Agent, with 8 offices.
+
+**Disclaimer: Actual costs will vary from customer to customer based on your product and implementation decisions**
+
+Assuming you implement the architecture model above, the cost calculation will approximate as below;
+
+#### **Month 1 seed calls**
+
+```
+p = initial seed of live properties for agent = 700
+i = average number of images per property = 15
+n = number of paged items per API call = 100
+
+Calculation: ((p / n) + ((p * i) / n) * b
+
+Initial seed calls: (700 / 100) + ((700 * 15) / 100 = 112 API Calls
+```
+
+#### **Rolling monthly calls**
+
+```
+p = average number of properties in database = 700
+i = average number of images changed per hour = 100 (30 actual, but anything up to 100 will be charged as 1 call if paging set to 100)
+c = average number of properties changed per hour = 100 (30 actual, but anything up to 100 will be charged as 1 call if paging set to 100)
+n = number of paged items per API call = 100
+d = number of days in a month running scheduled job = 31
+h = hours in day running scheduled job = 24
+
+
+Calculation Scheduled Calls Per Month: ((c / n) + ((p / n) * (i / n)) * h ) * d
+Calculation Monthly Housekeeping Calls: ((p / n) * b) * d
```

Comment:
`* b` doesn't need to be here

((properties in db / page size) * daysInMonth) = total